mainnet-images.bzl

```diff
@@ -0,0 +1,64 @@
+"""
+This module defines Bazel targets for the mainnet versions of ICOS images
+"""
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
+
+MAINNET_REVISION = "6968299131311c836917f0d16d0b1b963526c9b1"
+
+def base_download_url(git_commit_id, variant, update, test):
+    return "https://download.dfinity.systems/ic/{git_commit_id}/{variant}/{component}{test}/".format(
+        git_commit_id = git_commit_id,
+        variant = variant,
+        component = "update-img" if update else "disk-img",
+        test = "-test" if test else "",
+    )
+
+def mainnet_images():
+    """
+    Provides Bazel targets for mainnet ICOS images.
+    """
+
+    http_file(
+        name = "mainnet_setupos_disk_image",
+        downloaded_file_path = "disk-img.tar.zst",
+        url = base_download_url(MAINNET_REVISION, "setupos", False, False) + "disk-img.tar.zst",
+    )
+
+    http_file(
+        name = "mainnet_hostos_shas",
+        downloaded_file_path = "SHA256SUMS",
+        url = base_download_url(MAINNET_REVISION, "hostos", True, False) + "SHA256SUMS",
+    )
```

Comment:
Shouldn't you specify `sha256`s here to ensure hermiticity of the build? [Like I do here](https://github.com/dfinity/ic/pull/1468/files#diff-13270af6472df7268d3e162a8462748061a92ddbfd29f8f156bcb52813149646R16-R21).